contracts/SortitionPool.sol

```diff
@@ -95,6 +100,19 @@ contract SortitionPool is SortitionTree {
     }
   }
 
+  /// @notice Transfers the ownership of the sortition pool.
+  /// @dev Can be called only by the contract owner.
+  /// @param newOwner Address of the new pool owner.
+  function transferOwnership(address newOwner) public {
```

Comment:
We already have OpenZeppelin in dependencies. Let's update the version (we use quite an old one) and use `Ownable` for this. This would play nice with these two comments:
https://github.com/keep-network/sortition-pools/pull/132#discussion_r744562582
https://github.com/keep-network/sortition-pools/pull/132#discussion_r744560140

Reply:
See https://github.com/keep-network/sortition-pools/pull/131/commits/fc48f6271fbdb8fb2b61bfb19fa410a331986f27